client/data/disputes/hooks.ts

```diff
@@ -44,7 +49,7 @@ export const useDispute = (
 	const { acceptDispute } = useDispatch( STORE_NAME );
 	const doAccept = () => acceptDispute( id );
 
-	return { dispute, isLoading, doAccept };
+	return { dispute, isLoading, error, doAccept };
```

Comment:
Adding support for handling dispute load errors is possibly overengineering for this use case (fallback redirect), but is useful in general so let's keep it :) 

I do wonder how we got to this point in the project without adding this!

Reply:
The resolver shows a snackbar on error: “Error retrieving dispute”. I’d guess that this has been a sufficient way to handle errors until now.

Reply:
Right, so is there a chance we don't need this extra support? Happy for it to merge, just clarifying the value.

Reply:
Yep the value is low at the moment, we’re only checking for a fetch error in this component. Alternatively, we could move the error handling to the component, but I think it’s more predictable to handle dispute fetch errors in a way that is consistent with other wp.data resolvers.